### ISSUE TITLE
Invalid number of parameters in SUBSTITUTE statement

### ISSUE THREAD

**Issue by stefandrissen** [Riverside-Software/sonar-openedge#1075]
Rule does not deal with ampersand escaping:
````
message substitute( '&1 &&2':u, 'one':u ).
````
and incorrectly reports: 2 parameter(s) expected by SUBSTITUTE function, but 1 provided

**Comment by gquerret** [comment]
Confirmed

**Comment by gquerret** [comment]
Fixed in develop branch
